english/end/end.md

```diff
@@ -0,0 +1,8 @@
+Lol this is the dead-end page that I talked to you about ;)
+
+But actually this is just a joke. I hope you enjoyed the game :)
+
+Here is a couple of options that you will wanna pick:
+
+[If you wanna eat the marshmallow, please click here lol](../eatmarshmallow/eatmarshmallow.md)
+[If you want the DreamRide, please click here](../enjoy-DreamRide/enjoy-DreamRide.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.